config/main.py

```diff
@@ -728,7 +728,24 @@ def storm_control_delete_entry(port_name, storm_type):
     return True
 
 
-def _clear_qos():
+def _wait_until_clear(table, interval=0.5, timeout=30):
+    start = time.time()
+    empty = False
+    app_db = SonicV2Connector(host='127.0.0.1')
+    app_db.connect(app_db.APPL_DB)
+
+    while not empty and time.time() - start < timeout:
+        current_profiles = app_db.keys(app_db.APPL_DB, table)
+        if not current_profiles:
+            empty = True
+        else:
+            time.sleep(interval)
+    if not empty:
+        click.echo("Operation not completed successfully, please save and reload configuration.")
+    return empty
```

Comment:
No need to return here since the calling function does not handle the return value

Reply:
> No need to return here since the calling function does not handle the return value

@neethajohn it wasn't needed for the function call, but I used it on the unit tests.
Do you have any suggestion on how to test it without the return?